openbas-api/src/main/java/io/openbas/rest/exercise/ExerciseApi.java

```diff
@@ -945,5 +947,22 @@ public List<Endpoint> endpoints(@PathVariable String exerciseId) {
   public List<Document> documents(@PathVariable String exerciseId) {
     return this.documentService.documentsForSimulation(exerciseId);
   }
+
+  @GetMapping(EXERCISE_URI + "/{exerciseId}/scenario")
+  @RBAC(
+      resourceId = "#exerciseId",
+      actionPerformed = Action.READ,
+      resourceType = ResourceType.SIMULATION)
+  @Operation(summary = "Get the Scenario linked to the exercise")
```

Comment:
"Get the Scenario linked to the exercise" => "Get the Scenario linked to the simulation"